numpy/lib/type_check.py

```diff
@@ -271,6 +279,11 @@ def isreal(x):
     --------
     >>> np.isreal([1+1j, 1+0j, 4.5, 3, 2, 2j])
     array([False,  True,  True,  True,  True, False])
+    
+    Returns True for all elements in input array which are not of complex type. 
```

Comment:
"If there is at least one element of the input array that is not of numeric types" Isn't this the explanation you intended?

Reply:
yep, now its clearer since the example creates an array of dtype=object, and ive clarified that it returns True for all elements if so